libs/shared/lessons/src/lib/lesson-item/lesson-item.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import {
+  Typography,
+  ListItem,
+  Icon,
+  Popover,
+  Button,
+  ButtonGroup,
+  Link,
+} from '@material-ui/core';
+
+import './lesson-item.scss';
+
+export interface Lesson {
+  _id: string;
+  title: string;
+  datime: string;
+  creator?: {
+    _id: string;
+    name: string;
+    imageUrl?: string;
+  };
+  imageUrl: string;
+  description: string;
+}
+
+export interface Action {
+  title: string;
+  action(lesson: Lesson): void;
+}
+
+export const LessonItem = (props: { lesson: Lesson; actions: Action[] }) => {
+  const [anchorEl, setAnchorEl] = React.useState<HTMLButtonElement | null>(
+    null
+  );
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const open = Boolean(anchorEl);
+
+  const preventDefault = (event: React.SyntheticEvent) => {
+    event.preventDefault();
+  };
+
+  return (
+    <ListItem className="lesson" alignItems="flex-start">
+      <div className="lesson__details">
+        <Link className="lesson__title" href="#" onClick={preventDefault}>
+          <Typography variant="h3">{props.lesson.title}</Typography>
+        </Link>
+
+        <address className="author lesson__text">
+          By
+          <a rel="author" href="#">
+            {props.lesson.creator.name}
+          </a>
+        </address>
+        <time className=" lesson__text" dateTime={props.lesson.datime}>
+          {new Date(props.lesson.datime).toDateString()}
+        </time>
+        <span className=" lesson__text">2h30min</span>
```

Comment:
what is '2h30min'?

Reply:
Length of the lesson. But now back-end doesn't have a lesson end date, so I left how it is, or we can replace to something else. Not sure what ...